### PR TITLE
Add page skeleton and templates for views

### PR DESCRIPTION
**Pull request by karol22**

### CODE REVIEW

**Comment by astrobliss** [comment]
oof we should've made this pull request a while ago, thanks for putting it through @karol22. It looks good to me though.

File: src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        String redirectURL;
+        if (userService.isUserLoggedIn()) {
+            redirectURL = destinationURL;
+        } else {
+            redirectURL = userService.createLoginURL(destinationURL);
+        }
+        response.sendRedirect(redirectURL);
+    }
+
+    private String getDestinationUrl(HttpServletRequest request) {
```
Comment (the-guti) [inline src/main/java/com/google/sps/servlets/LoginServlet.java:36]: Redundant method with logout service

File: src/main/java/com/google/sps/servlets/LogoutServlet.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        String redirectURL;
+        if (userService.isUserLoggedIn()) {
+            redirectURL = userService.createLogoutURL(destinationURL);
+        } else {
+            redirectURL = destinationURL;
+        }
+        response.sendRedirect(redirectURL);
+    }
+
+    private String getDestinationUrl(HttpServletRequest request) {
```
Comment (astrobliss) [inline src/main/java/com/google/sps/servlets/LogoutServlet.java:36]: use Requests.getParameter(...)

Comment (the-guti) [inline src/main/java/com/google/sps/servlets/LogoutServlet.java:36]: Redudant method with login service

File: src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 1 above)
Comment (astrobliss) [inline src/main/java/com/google/sps/servlets/LoginServlet.java:36]: use Requests.getParameter(..)

File: src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
```
Comment (astrobliss) [inline src/main/java/com/google/sps/servlets/LoginServlet.java:3]: Later Standardize Import Order.

Add TODO to Readme.

File: src/main/java/com/google/sps/utils/Requests.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            return defaultValue;
+        }
+    }
+
+    /**
+     * Returns true iff the request contains the parameter parameterName, its value is ignored
+     */
+    public static boolean hasParameter(HttpServletRequest request, String parameterName) {
+        String parameterValue = request.getParameter(parameterName);
+        return parameterValue == null;
```
Comment (astrobliss) [inline src/main/java/com/google/sps/utils/Requests.java:27]: should be =!

Comment (the-guti) [inline src/main/java/com/google/sps/utils/Requests.java:27]: Should be not !=

File: src/main/webapp/assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+}
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
+  line-height: 60px; /* Vertically center the text there */
+  background-color: #f5f5f5;
+}
```
Comment (the-guti) [inline src/main/webapp/assets/css/style.css:17]: Empty carriage at the end

File: src/main/webapp/assets/html/footer.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+    <div class="container">
+    <span class="text-muted">Made by Team #6. Google SPS Summer 2020. Contribute on <a href="https://github.com/axgtz/summer20-sps-team6">Github</a></span>
```
Comment (the-guti) [inline src/main/webapp/assets/html/footer.html:3]: indent?

File: src/main/webapp/create.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    <div id="header"></div>
+    <h1>
+        Create new playlist
+    </h1>
+    <p>
+        Search for your favorite songs:
+    </p>
+
+
+    <script src="assets/js/script.js"></script>
```
Comment (the-guti) [inline src/main/webapp/create.html:33]: move them to bottom with rest of scripts

File: src/main/webapp/template.html [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+        Template page
+    </h1>
+    <p>
+        This is an example
+    </p>
+
+
+    <script src="assets/js/script.js"></script>
+    <script src="https://sdk.scdn.co/spotify-player.js"></script>
+    <script src="assets/js/spotify.js"></script>
```
Comment (the-guti) [inline src/main/webapp/template.html:35]: move scripts to bottom with the rest

**Review by astrobliss — APPROVED** [review]

**Review by the-guti — APPROVED** [review]

**Pull request merged**